### PR TITLE
Add GPS functionality for Android

### PR DESCRIPTION
**Pull request by skysphr**
This adds four GPS-related functions to the Input singleton:
- start_gps_tracker() where the user specifies the minimum frequency of location fixes and the minimum distance that the device has to move in order to get a location update (for battery saving reasons)
- stop_gps_tracker()
- get_locatino() which gives a Lat/Long Vector2 of the location
- get_altitude() which gives a float representing altitude above the WGS84 ellipsoid

### CODE REVIEW

File: core/input/input.cpp [hunk 1]
```diff
@@ -121,6 +121,10 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_accelerometer"), &Input::get_accelerometer);
 	ClassDB::bind_method(D_METHOD("get_magnetometer"), &Input::get_magnetometer);
 	ClassDB::bind_method(D_METHOD("get_gyroscope"), &Input::get_gyroscope);
+	ClassDB::bind_method(D_METHOD("start_gps_tracker", "time_ms", "distance"), &Input::start_gps_tracker);
+	ClassDB::bind_method(D_METHOD("stop_gps_tracker"), &Input::stop_gps_tracker);
+	ClassDB::bind_method(D_METHOD("get_location"), &Input::get_location);
+	ClassDB::bind_method(D_METHOD("get_altitude"), &Input::get_altitude);
```
Comment (Calinou) [inline core/input/input.cpp:127]: Remember to run `--doctool` with a compiled editor binary to [update the class reference XML](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html), then add a description in `doc/classes/Input.xml` for the newly exposed methods.

Comment (skysphr) [reply core/input/input.cpp:127]: Added, thanks.

**Comment by m4gr3d** [comment]
@skysphr This sort of functionality belongs in external Godot plugins and not in core.

**Comment by m4gr3d** [comment]
@skysphr For reference take a look at the documentation for Android plugin: https://docs.godotengine.org/en/stable/tutorials/plugins/android/android_plugin.html

**Pull request closed without merging**